chain/sync.go

```diff
@@ -528,8 +528,16 @@ func blockSanityChecks(h *types.BlockHeader) error {
 
 // Should match up with 'Semantical Validation' in validation.md in the spec
 func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) error {
+	validationStart := time.Now()
+	defer func() {
+		durationMillisec := float64(time.Since(validationStart).Round(time.Microsecond).Nanoseconds()) / float64(time.Millisecond)
+		stats.Record(ctx, metrics.BlockValidationDurationMilliseconds.M(durationMillisec))
+		log.Infow("block validation", "duration_ms", durationMillisec, "block_height", b.Header.Height)
```

Comment:
we usualy use do just `"took", duration`

Reply:
No prob. `{"level":"info","ts":"2020-06-04T23:25:48.211Z","logger":"chain","caller":"chain/sync.go:535","msg":"block validation","took":254.352,"height":"72754"}`

Reply:
It should probably be in seconds, so it is consistent. I've sent suggestions below.